src/app/identity.service.ts

```diff
@@ -152,7 +153,7 @@ export class IdentityService {
   isTestnet = false;
 
   constructor(private tracking: TrackingService) {
-    window.addEventListener("message", (event) => this.handleMessage(event));
+    // window.addEventListener("message", (event) => this.handleMessage(event));
```

Comment:
should I delete it?

Reply:
This should be reverted, we want to keep it. I must have commented it to test something.